### PR TITLE
Becatrue/187/config json

### LINKED ISSUE #187: Change the name "setup.json" -> "config.json"
Since there is "setup.py" for completely different purpose, the name is confusing.
Therefore, I want to change the file name to "config.json" and the variable names in the codes as well.

### PR DESCRIPTION
**Pull request by BECATRUE**
This PR will close #187.

I just renamed `setup.json` to `config.json`.

### CODE REVIEW

**Comment by kangz12345** [comment]
Aren't there any comments, for example how-to-use?

File: qiwis.py [hunk 1]
```diff
@@ -667,7 +667,7 @@ def _get_argparser() -> argparse.ArgumentParser:
         description="QuIqcl Widget Integration Software"
     )
     parser.add_argument(
-        "-s", "--setup", dest="setup_path", default="./setup.json",
+        "-s", "--setup", dest="setup_path", default="./config.json",
```
Comment (kangz12345) [inline qiwis.py]: Let's change the argument name to --config.

**Review by kangz12345 — CHANGES_REQUESTED** [review]

File: qiwis.py (hunk 1 above)
Comment (BECATRUE) [reply qiwis.py]: It looks clear! I will apply it.

**Comment by BECATRUE** [comment]
I will fix the unittest errors😅

**Review by kangz12345 — DISMISSED** [review]
LGTM

File: .github/workflows/unittest.yml [hunk 2]
```diff
@@ -25,5 +25,5 @@ jobs:
         sudo apt-get install python3-pyqt5
     - name: Unittest the code with coverage
       run: |
-        xvfb-run `which coverage` run -m test
-        xvfb-run `which coverage` report -m
```
Comment (kangz12345) [inline .github/workflows/unittest.yml:29]: ??? Why didn't we figure this out 😆

**Review by kangz12345 — APPROVED** [review]
LGTM

**Comment by kangz12345** [comment]
Please merge this after @Aijuh reviews this.

File: .github/workflows/unittest.yml (hunk 2 above)
Comment (BECATRUE) [reply .github/workflows/unittest.yml:29]: Haha😃

I also realized now.. but it didn't make any problem as far!
I just update this to follow other projects' rule.

Comment (BECATRUE) [reply .github/workflows/unittest.yml:29]: Haha😃

I also realized now.. but it didn't make any problem as far!
I just updated this to follow other projects' rule.

Comment (kangz12345) [reply .github/workflows/unittest.yml:29]: Oh I see 👍

**Review by Aijuh — APPROVED** [review]
I didn't know that this PR needs my review.
As far as I can check, there seems no problem.

**Comment by BECATRUE** [comment]
> I didn't know that this PR needs my review. As far as I can check, there seems no problem.

Thanks for approving!

Until everyone submits his or her own first PR, Jiyong and I will merge it although there is only one approve.
However, if you afford to leave reviews, reviewing may help you to understand codes!

**Pull request merged**